.github/workflows/deploy-book.yml

```diff
@@ -378,6 +408,38 @@ jobs:
           echo "No build errors or warnings encountered." >> summary.md
         fi
 
+    - name: Add a custom 404 to redirect
+      env:
+        # https:(0)/(1)/uname.github.io(2)/rname(3)/bname(4) */
+        # if urlParts.len < 4, bname is undefined */
```

Comment:
Am I understanding correctly that the redirecting doesn't work deeper than for example this page? https://teachbooks.github.io/testable-template/2024/some_content/text_and_code.html

Would be nice if there's no limit on the 'deepness'

Reply:
There should be no limit to depth as of now -- the redirect procedure is really simple -- for `abc/x/y/.../z` to `2024/abc/x/y/.../z` given `abc` is not a branch name